packages/cli/src/commands/dev-contracts.ts

```diff
@@ -15,6 +15,7 @@ import { Address } from "viem";
 const devOptions = {
   rpc: deployOptions.rpc,
   configPath: deployOptions.configPath,
+  alwaysPostDeploy: deployOptions.alwaysPostDeploy,
```

Comment:
"always post deploy" kinda sounds like we're "posting" the deploy somewhere

maybe this should be `alwaysRunPostDeploy`?